contentcuration/contentcuration/templates/settings/storage.html

```diff
@@ -143,46 +143,68 @@ <h5>{% trans "Request More Space" %}</h5>
 
 <script>
 	window.addEventListener('load', function(){
-		$('[data-toggle="tooltip"]').tooltip();
 
 		// Allow deselection if item is clicked, don't deselect everything on click
-		$(".multi-select-field").on("mousedown", function(ev) {
-			ev.preventDefault();
-			ev.target.selected = !ev.target.selected;
-
-			// Prevent select box from jumping to the top every click
-			var st = this.scrollTop;
-			var self = this;
-			setTimeout(function() {
-				self.scrollTop = st, 0;
+		Array.from(document.getElementsByClassName('multi-select-field')).forEach(field => {
+			field.addEventListener('mousedown', event => {
+				event.preventDefault();
+				event.target.selected = !event.target.selected;
+
+				// Prevent select box from jumping to the top every click
+				var st = field.scrollTop;
+				setTimeout(() => {
+					field.scrollTop = st, 0;
```

Comment:
I know this was existing but this is buggish-looking code. It's reminiscent of `scrollTo(x, y)` but backwards. I think calling that is preferred to setting `scrollTop`, but regardless the `, 0` is useless as it is. There are duplicated occurrences of this oddness.